### DISCUSSION TITLE
Interactive documents

### DISCUSSION THREAD

**Discussion by sszlime00** [unocss/unocss#3109] [Q&A]
### Clear and concise description of the problem

How to upload custom presets to unocss interactive documents

### Suggested solution

Specific steps for uploading

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/unocss/unocss/blob/main/CONTRIBUTING.md).
- [X] Read the `README.md` of using the package.
- [X] Already used the [Interactive Docs](https://unocss.dev/interactive/).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by zyyv** [comment]
Refer to my screenshot. Custom your uno configuration.

<img width="1144" alt="image" src="https://github.com/unocss/unocss/assets/42139754/e3a0103b-03ba-4ca2-a1b5-2f2929f17a01">
<img width="736" alt="image" src="https://github.com/unocss/unocss/assets/42139754/44fd443c-8163-4935-8d0d-a87b0143f0ee">
